### PR TITLE
Updating webhook docs

### PR DESCRIPTION
**Pull request by IaroslavTitov**
### Proposed changes
- Updating webhook page to include new environment webhooks (not released yet) and webhook groups
- Not merging this in until release time

### Testing
- Read through the generated website, confirmed it looks good

### CODE REVIEW

**Comment by IaroslavTitov** [comment]
I really hate this:
image
Is there any way to control width of columns or whole table? My screen has plenty of space

**Comment by interurban** [comment]
hey @IaroslavTitov ; @sean1588 is looking into if we can make this cleaner; hugo takes the markdown table and auto converts it into some html table and we are limited by the width of the page. But he's going to test out a possible easy fix now.

**Comment by sean1588** [comment]
I tried to get hugo to force some text wrapping in the first column, and hopefully get the other ones will layout better since they will have more space, since there seems to be just a few line items that are long in the first column that are causing it to take up so much horizontal space. It didn't help that much though.

However I tried toggling the screen width using the widget that we have in the upper right of the top nav and it looks pretty good when the screen width is toggled to make it wider. This is how it looks with the width toggled.

Screen Shot 2024-09-12 at 12 23 17 PM

File: content/docs/pulumi-cloud/webhooks.md [hunk 1]
```diff
@@ -128,64 +130,70 @@ class PulumiServiceWebhook: Stack
...
+| `drift_detection_failed`      		| `drift_detection`   				| Organization or Stack      | When a drift detection run fails.                |
+| `drift_remediation_succeeded` 		| `drift_remediation` 				| Organization or Stack      | When a drift remediation run succeeds.           |
+| `drift_remediation_failed`    		| `drift_remediation` 				| Organization or Stack      | When a drift remediation run fails.              |
+
+And this table describes the various filter groups available to easily subscribe to all events within a group.
+
+| Group			    | Event Kinds Included 																									|
+|---------------|-----------------------------------------------------------------------|
+|`stacks`		    |`stack`, `stack_preview`, `stack_update`																|
+|`deployments`	|`deployment`, `drift_detection`, `drift_remediation`										|
```
Comment (dschaller) [inline content/docs/pulumi-cloud/webhooks.md:186]:
```suggestion
| Group                | Event Kinds Included                                                       |
|----------------|------------------------------------------------------|
| `stacks`            | `stack`, `stack_preview`, `stack_update`                   |
| `deployments` | `deployment`, `drift_detection`, `drift_remediation` |
```

File: content/docs/esc/webhooks.md [hunk 2]
```diff
@@ -0,0 +1,414 @@
+---
+title_tag: "ESC Webhooks"
+meta_desc: ESC Webhooks allow you to notify external services of events happening within your Pulumi organization. Learn how to create and manage webhooks here.
```
Comment (dschaller) [inline content/docs/esc/webhooks.md:3]:
```suggestion
meta_desc: ESC Webhooks allow you to notify external services of events happening within your Pulumi environments. Learn how to create and manage webhooks here.
```

File: content/docs/esc/webhooks.md [hunk 3]
```diff
@@ -0,0 +1,414 @@
...
+- /docs/esc/webhooks/
+---
+
+{{% notes "info" %}}
+ESC Webhooks is a feature available on the Pulumi Team, Enterprise and Business Critical editions.
+To try it out, start a [trial](https://app.pulumi.com/site/trial) now.
+{{% /notes %}}
+
+ESC Webhooks allow you to notify external services of events
+happening within your Pulumi organization. For example,
```
Comment (dschaller) [inline content/docs/esc/webhooks.md:25]:
```suggestion
ESC Webhooks allow you to notify external services of events
happening within your ESC environments. For example,
```

File: content/docs/esc/webhooks.md [hunk 4]
```diff
@@ -0,0 +1,414 @@
...
+you can trigger a notification whenever a new revision of an environment is created.
+Whenever an event occurs, Pulumi will send an HTTP `POST` request to
```
Comment (dschaller) [inline content/docs/esc/webhooks.md:27]:
```suggestion
When an event occurs, Pulumi will notify the registered webhook listeners via a HTTP `POST` request with metadata about the event
```

File: content/docs/esc/webhooks.md [hunk 5]
```diff
@@ -0,0 +1,414 @@
...
+all registered webhooks. The webhook can then be used to emit a
```
Comment (dschaller) [inline content/docs/esc/webhooks.md:28]:
```suggestion
When an event occurs, Pulumi will notify the registered webhook listeners via a HTTP `POST` request with metadata about the event. The webhook can then be used to emit a
```

File: content/docs/esc/webhooks.md [hunk 6]
```diff
@@ -0,0 +1,414 @@
...
+notification, start running integration tests, or even update Pulumi stacks.
+
+Webhooks can be used for pretty much anything you want, and are the foundation
```
Comment (dschaller) [inline content/docs/esc/webhooks.md:31]:
```suggestion
There are large number of real life applications for webhooks including serving as the foundation```

File: content/docs/esc/webhooks.md (hunk 5 above)
Comment (IaroslavTitov) [reply content/docs/esc/webhooks.md:28]: This one is copy-pasted from original page. Should I change both or leave it be? I think we want to stay in sync

File: content/docs/pulumi-cloud/webhooks.md (hunk 1 above)
Comment (IaroslavTitov) [reply content/docs/pulumi-cloud/webhooks.md]: It was straight in my editor :/

File: content/docs/esc/webhooks.md (hunk 6 above)
Comment (IaroslavTitov) [reply content/docs/esc/webhooks.md:31]: Same here, copypasta from original page

**Review by seanyeh — APPROVED** [review]

File: content/docs/esc/webhooks.md [hunk 7]
```diff
@@ -0,0 +1,414 @@
...
+of most _ChatOps_ workflows.
+
+## Overview
+
+ESC Webhooks can be attached to either a environment or an organization. Environment webhooks
+will be notified of events specific to the environment. Organization
+webhooks will be notified for events happening within each of the organization's
+environments.
+
+The Webhooks page for organization is under Organization Settings tab, and for Environments is a tab on each Environment page
```
Comment (dschaller) [inline content/docs/esc/webhooks.md:41]:
```suggestion
Organization webhooks can be managed on the Organization Settings page. Environment webhooks can be managed from the webhooks tab on each Environment's detail page.
```

File: content/docs/esc/webhooks.md [hunk 8]
```diff
@@ -0,0 +1,414 @@
...
+| `environment_tag_created`				    | `environment_tag`					      | Organization or Environment| When a new environment tag is created.           |
+| `environment_tag_deleted`				    | `environment_tag`					      | Organization or Environment| When an environment tag is deleted.              |
+| `environment_tag_updated`				    | `environment_tag`					      | Organization or Environment| When an environment tag is updated.              |
+| `imported_environment_changed`		  | `imported_environment_changed`	| Organization or Environment| When an imported environment was changed.        |
+
+There is also an `environments` filter group that lets you easily subscribe to all environment events.
+
+| Group			    | Event Kinds Included 																									                                              |
+|---------------|---------------------------------------------------------------------------------------------------------------------|
+|`environments`	|`environment`, `environment_revision`, `environment_revision_tag`, `environment_tag`, `imported_environment_changed`	|
```
Comment (dschaller) [inline content/docs/esc/webhooks.md:179]: I think you might have used tabs here instead of spaces

File: content/docs/esc/webhooks.md (hunk 5 above)
Comment (dschaller) [reply content/docs/esc/webhooks.md:28]: either way works

**Review by dschaller — APPROVED** [review]

File: content/docs/esc/webhooks.md (hunk 8 above)
Comment (IaroslavTitov) [reply content/docs/esc/webhooks.md:179]: Yep, I did😩 
I'll redo into spaces

File: content/docs/esc/webhooks.md [hunk 9]
```diff
@@ -0,0 +1,414 @@
...
+menu:
+  pulumiesc:
+    weight: 9
+aliases:
+- /docs/intro/environment/webhooks/
+- /docs/intro/environments/webhooks/
+- /docs/intro/esc/webhooks/
+- /docs/environment/webhooks/
+- /docs/environments/webhooks/
+- /docs/esc/webhooks/
```
[the remark this answers is not in the document]
Comment (IaroslavTitov) [reply content/docs/esc/webhooks.md:16]: Ah, ack, I thought these were for people to find it easier

**Pull request merged**